lib/diagnostic.rb

```diff
@@ -5,6 +5,7 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
+    list.max_by{ &:length }
```

Comment:
On the right path. Since it's a comparison, we need to use the shapeship operator.

`list.max { |a, b| a.length <=> b.length }`